### PR TITLE
Add link to team F

### PR DESCRIPTION
**Pull request by yousifm**
Add team F link to Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ Hello from CUHK CSCI3251! This is another playground for our course students to
 There are some teams in our course project:
 
 * [Team X](https://csci3251-2020.github.io/project-team-x/ 'Team-X') _@chuckjee_ (Please remove this line when you add the first actual team below)
+* [Team F](https://csci3251-2020.github.io/project-team-f/ 'Team-F')
```
Comment (chuckjee) [inline README.md]: Pretty good. Please sign your username like in the demo line. (and read carefully)

**Review by chuckjee — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yousifm) [reply README.md]: Fixed. Thanks

**Review by chuckjee — APPROVED** [review]
Great work

**Review by chuckjee — APPROVED** [review]
Great work

**Pull request merged**